### DISCUSSION TITLE
How to use custom domain after verifying

### DISCUSSION THREAD

**Discussion by UtterDonkey** [community/community#77314] [Other Feature Feedback, Questions, & Ideas] [answered]
I have verified my custom domain name however when I try to set it as the custom domain on a repository, I get "The custom domain `my domain` is already taken. If you are the owner of this domain, check out https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages for information about how to verify and release this domain."

I'm unable to find anything about what to actually do after verifying my domain.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! 

Has it been more than seven days? The process to release the domain from its current location takes 7 days to complete.

**Reply by drbruced12** [reply]
Thanks, I have this issue too. But I don't understand what is going on. I was able to verify domains for a couple of my repositories in no time at all, and then I updated my DNS settings to point to GitHub, and it all worked in minutes. But now that I am doing this for the third time, after verifying the new domain, I have run into the "wait 7 days" issue. I don't know what it means to "release the domain from its other location". Any insights?

**Reply by queenofcorgis** [reply]
@drbruced12 are you saying this domain has never been used previously?

**Reply by drbruced12** [reply]
Well, I am trying to understand what it means for a domain to be "in use". Like other domains that I have, the one I am trying to configure currently points to another (non-GitHub) website. So I have never used it within GitHub before, but it is in use. But I am confused because the first two times I tried to configure a custom domain, it was also being used outside GitHub for an existing website, and there was no 7-day wait.

**Reply by drbruced12** [reply]
An update here: I did find that another organization was using the domain that I was trying to configure for my GitHub pages. The 7-day waiting period has now passed (they didn't actually want to use it for anything) and my issue is resolved. The reason i didn't hit this previously was because those prior domains, while also "in use" (in the sense that they pointed to other web sites), were not being used by anyone else within GitHub. So I've answered my own question.

**Reply by queenofcorgis** [reply]
Thank you for coming back to share your experience and what the culprit was @drbruced12 !

**Comment by npskelakam** [comment]
Hi there, I am also facing similar issues. So you are saying that I should wait 7 days after verifying?

**Reply by UtterDonkey** [reply]
Yes, that was the solution for me.
